src/components/filter/MessagesFilterPanel.tsx

```diff
@@ -282,6 +284,15 @@ const MessagesFilterPanel = () => {
 			<MessageReplayModal />
 			<MessageFilterWarning />
 			<MessagesFilterSessionFilter config={sessionFilterConfig} submitChanges={submitChanges} />
+			{true && (
+				<MessageExport
+					isExport={messagesStore.isExport}
+					enableExport={() => messagesStore.enableExport()}
```

Comment:
you can pass a function reference directly without wrapping it with another function

`enableExport={messagesStore.enableExport}`

Reply:
If I don't wrap it then there is error because 'this' in 'enableExport' and 'disableExport' is 'undefined'.

Reply:
That's why i prefer using arrow functions inside stores as well. if you convert disableExport, enableExport, endExport to be arrow functions you can avoid wrapping them